test/lib/mayaUsd/nodes/testProxyShapeConversion.py

```diff
@@ -50,9 +50,11 @@ def setUp(self):
         """Clear the scene and setup the Maya preferences."""
         cmds.file(f=True, new=True)
         # Make sure up-axis is Z.
-        cmds.upAxis(axis='z')
+        if cmds.upAxis(query=True, axis=True).lower() != 'z':
```

Comment:
Do these occur when reloading the plugin?

Reply:
The commands were issuing warnings that the up-axis was already Z, I'm just reducing the amount of noise when running the tests.